src/TConsole.cpp

```diff
@@ -583,6 +583,9 @@ void TConsole::resizeEvent(QResizeEvent* event)
     int x = event->size().width();
     int y = event->size().height();
 
+    // don't call event in lua if size didn't change
+    bool preventLuaEvent = (x == oldSize.width() && y == oldSize.height());
```

Comment:
Surely this is simpler but equivalent?
```suggestion
    bool preventLuaEvent = (event->size() == oldSize);
```
though renaming it to `doLuaEvent` ***and inverting*** the logic might be a little clearer for the human brain to parse.